packages/web/src/scss/components/UNSTABLE_Footer/README.md

```diff
@@ -0,0 +1,231 @@
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+            </ul>
+          </nav>
+
+          <nav aria-labelledby="footer-part-two" class="mb-600 mb-tablet-800">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-two">Section headline</h3>
+
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+            </ul>
+          </nav>
+
+          <nav aria-labelledby="footer-part-three" class="mb-600 mb-tablet-800">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-three">Section headline</h3>
+
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+            </ul>
+          </nav>
+
+          <nav aria-labelledby="footer-part-four" class="mb-600 mb-tablet-800">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-four">Section headline</h3>
+
+            <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+
+              <li>
+                <a href="www.example.com"> Link </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <ul class="UNSTABLE_Footer__copyright pt-900">
+        <li>
+          <a href="www.example.com" class="link-secondary"> Legal notice </a>
+        </li>
+
+        <li>
+          <a href="www.example.com" class="link-secondary"> Terms of service </a>
+        </li>
+
+        <li>
+          <a href="www.example.com" class="link-secondary"> Privacy policy </a>
+        </li>
+
+        <li>
+          <a href="www.example.com" class="link-secondary"> Manage cookies </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</footer>
+```
+
+### Columns variation
```

Comment:
maybe this title and the next paragraph should mention that you are talking only about the "links" section.